hack/windows/Prepare-Node.ps1

```diff
@@ -112,18 +152,25 @@ Remove-Item -Force .\nssm.zip
 
 $env:path += ";$global:NssmInstallDirectory"
 $newPath = "$global:NssmInstallDirectory;" +
-        [Environment]::GetEnvironmentVariable("PATH", [EnvironmentVariableTarget]::Machine)
+[Environment]::GetEnvironmentVariable("PATH", [EnvironmentVariableTarget]::Machine)
 
 [Environment]::SetEnvironmentVariable("PATH", $newPath, [EnvironmentVariableTarget]::Machine)
 
 Write-Host "Registering kubelet service"
 nssm install kubelet $global:Powershell $global:PowershellArgs $global:StartKubeletScript
-nssm set kubelet DependOnService docker
+
+if ($ContainerRuntime -eq "Docker") {
+    nssm set kubelet DependOnService docker
+} elseif ($ContainerRuntime -eq "containerd") {
+    nssm set kubelet DependOnService containerd
```

Comment:
Does it work "nssm set kubelet DependOnService $ContainerRuntime" after modifying "Docker" to "docker"? Then we don't need this if-else

Reply:
changed and verified

Reply:
I mean using `nssm set kubelet DependOnService $ContainerRuntime` instead of if .... elseif ....

Reply:
Okay got your point